lib/.eslintrc

```diff
@@ -1,8 +1,13 @@
 {
   "parserOptions": {
+    "ecmaVersion": 2019,
```

Comment:
This seems fine to me, but I'm not sure what the default is here? If it defaults to latest, we should probably leave it unspecified.

Reply:
I reverted here since it seems that we supported `try {} catch {}` syntax now.
(Maybe duo to [.eslintrc.json](https://github.com/electron/electron/blob/master/.eslintrc.json) in the root dir)